index.js

```diff
@@ -172,29 +222,38 @@ module.exports = function (homebridge) {
       this.services.push(informationService);
 
       switch (this.config.type) {
+        case 'Dimmer':
+          let dimmerService = new homebridge.hap.Service.Lightbulb(this.config.name);
+          dimmerService
+            .getCharacteristic(homebridge.hap.Characteristic.On)
+            .on('get', this.getPowerState.bind(this))
+            .on('set', this.setPowerState.bind(this));
+          dimmerService
+            .getCharacteristic(homebridge.hap.Characteristic.Brightness)
+            .on('get', this.getDimLevel.bind(this))
+            .on('set', this.setDimLevel.bind(this));
+          this.services.push(dimmerService);
+          break;
+
         case 'TemperatureSensor':
-          let temperatureSensorService = new homebridge.hap.Service.TemperatureSensor(this.config.deviceId);
+          let temperatureSensorService = new homebridge.hap.Service.TemperatureSensor(this.config.name);
```

Comment:
The name parameter is the name with which the accessory will be displayed by default by Homekit. I didn't find "outlet0" as informative as "Living room outlet".